roles/ansible/tower/manage-credential-types/library/rest_get.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
```

Comment:
Same comment as for `list-dict-diff.py` above

Reply:
Done